.github/workflows/build-and-test.yaml

```diff
@@ -1,6 +1,10 @@
 name: build-and-test-scheduled
 
 on:
+  push:
+    branches:
+      - main
+      - tier4/proposal
```

Comment:
Please tell me about the database of CoveCov.
Are the results stored for branches individually?

Reply:
Yes., according to the documentation,
* Branches will show up when a commit is uploaded on that branch.
* Pull requests are synced, regardless of having a coverage report uploaded. The sync will occur upon the first pull request webhook sent from GitHub to Codecov.
* Codecov only shows commits that have uploaded coverage reports and are six months or less old.

https://docs.codecov.com/docs/frequently-asked-questions#where-are-all-my-commitsbranchespulls
https://docs.codecov.com/docs/frequently-asked-questions#where-are-my-older-commits-my-project-dashboard-doesnt-show-any-commit-data-in-the-line-graph

Reply:
Thank you.
What do you mean by `Pull requests are synced`?

Reply:
Maybe CodeCov is synchronizing pull requests in order to be able to use pull request related features:

https://docs.codecov.com/docs/pull-request-comments
https://docs.codecov.com/docs/commit-status#excluding-tests-example
https://docs.codecov.com/docs/github-checks

Reply:
Mm, what's `synchronizing`?

Reply:
> Mm, what's `synchronizing`?

The documentation doesn't have more information about the word "synced", I assume CodeCov needs to track pull requests in order to provide the Pulls tab in the dashboard.

ex.
https://app.codecov.io/gh/KeisukeShima/autoware.universe/pulls?page=1&state=open&order=-pullid

Reply:
Hmm, it's difficult... But thank you for your explanations! :pray: